app.py

```diff
@@ -57,6 +57,36 @@ def generate_message(self, now: datetime) -> Optional[str]:
             return None
 
 
+def generate_week_message(events: List[Event], tz: tzinfo) -> List[str]:
+    open_events = list(filter(lambda e: e.title.lower() == "open", events))
+    other_events = list(
+        filter(lambda e: e.title.lower() != "open", events))
+
+    messages = []
+
+    if len(open_events) > 0:
+        open_message = "今週の開館日です！\n"
+        for open in open_events:
+            date = open.start.astimezone(tz).date().strftime("%m/%d")
+            start = open.start.astimezone(tz).time().strftime("%H:%M")
+            end = open.end.astimezone(tz).time().strftime("%H:%M")
+            open_message += f"{date} {start}〜{end}\n"
+        open_message += "\nみなさんのお越しをお待ちしています!!"
+        messages.append(open_message)
+
+    if len(other_events) > 0:
+        other_message = "今週のイベント情報です！\n"
+        for event in other_events:
+            date = event.start.astimezone(tz).date().strftime("%m/%d")
+            start = event.start.astimezone(tz).time().strftime("%H:%M")
+            end = event.end.astimezone(tz).time().strftime("%H:%M")
+            other_message += f"{event.title} {date} {start}〜{end}\n"
+        other_message += "\nみなさんのお越しをお待ちしています!!"
```

Comment:
IMO: 
```suggestion
        other_message += "\nお申し込みの上ご参加ください。
        other_message += "\nみなさんのお越しをお待ちしています!!"
```
申し込んでほしいと一言あってもよいかもです．